### ISSUE TITLE
How to train the HMDB51 dataset to get the results of the paper?

### ISSUE THREAD

**Issue by yuer5823262** [MCG-NJU/TDN#8]
python -m torch.distributed.launch --master_port 12347 --nproc_per_node=3 \
            main.py  hmdb51  RGB --arch resnet50 --num_segments 16 --gd 20 --lr 0.001 \
            --lr_scheduler step --lr_steps  10 20 --epochs 25 --batch-size 4 \
            --wd 5e-4 --dropout 0.8 --consensus_type=avg --eval-freq=1 -j 4 \
            --tune_from='checkpoint/best.pth.tar'

### **Use this command, I did not get the approximate result of the paper. Can you share how you did it, Thank you very much**

**Comment by yztongzhan** [comment]
Thanks!  We use the following script for fine-tuning:
```
python -m torch.distributed.launch --master_port 12347 --nproc_per_node=8 \
main.py hmdb51 RGB --arch resnet50  --num_segments 16 --gd 20 --lr 0.02 \
--gd 20 --lr 0.002 --lr_steps 10 20 --epochs 25 --batch-size 16 --dropout 0.8 \
--consensus_type=avg --eval-freq=1 --j 4 --tune_from='checkpoint/best.pth.tar' --npb 
```
Then we perform 10-clip and 3-crop for accuracy evaluation protocol following [TEA](https://arxiv.org/abs/2004.01398) and [STM](https://arxiv.org/abs/1908.02486).
Considering that you only use 3 GPUs for this experiment, we recommend that you modify your learning rate according to to the [Linear Scaling Rule](https://arxiv.org/abs/1706.02677)

**Comment by weizh588** [comment]
> 

where is the hmdb51 checkpoint?

**Comment by weizh588** [comment]
> @yuer5823262 Thanks! We use the following script for fine-tuning:
> 
> ```
> python -m torch.distributed.launch --master_port 12347 --nproc_per_node=8 \
> main.py hmdb51 RGB --arch resnet50  --num_segments 16 --gd 20 --lr 0.002 \
> --lr_steps 10 20 --epochs 25 --batch-size 16 --dropout 0.8 --consensus_type=avg \
> --eval-freq=1 -j 4 --tune_from='checkpoint/best.pth.tar' --npb 
> ```
> 
> Then we perform 10-clip and 3-crop for accuracy evaluation protocol following [TEA](https://arxiv.org/abs/2004.01398) and [STM](https://arxiv.org/abs/1908.02486). Considering that you only use 3 GPUs for this experiment, we recommend that you modify your learning rate according to the [Linear Scaling Rule](https://arxiv.org/abs/1706.02677)

where is the hmdb51 checkpoint?
